storage/innobase/fil/fil0fil.cc

```diff
@@ -821,6 +821,23 @@ class Fil_shard {
     mutex_release();
   }
 
+  void clear_ibts() {
+    mutex_acquire();
+
+    for (auto it : m_ibt_deleted) {
+      auto space = it.second;
```

Comment:
Likewise
```suggestion
      const auto& space = it.second;
```

Reply:
this didn't work because fil_space_free() is expecting non const space, it frees object inside space_t* and also space to nullptr

Reply:
It doesn't accept reference "&" too.

Binding value of type 'fil_space_t *const' to reference to type 'fil_space_t *' drops 'const' qualifier  fil0fil.cc:1186:44: note: pass832,11         6%

Reply:
this looks strange to me

Reply:
@percona-ysorokin ok to revert to original?

Reply:
Yes, @satya-bodapati , here the original version is fine

Reply:
Thanks yura